src/main/scala/subsystem/RocketSubsystem.scala

```diff
@@ -30,7 +31,8 @@ case object RocketCrossingKey extends Field[Seq[RocketCrossingParams]](List(Rock
 trait HasRocketTiles extends HasTiles
     with CanHavePeripheryPLIC
     with CanHavePeripheryCLINT
-    with HasPeripheryDebug { this: BaseSubsystem =>
+    with HasPeripheryDebug {
+  this: BaseSubsystem =>
```

Comment:
This formatting doesn't match how we usually express self types; on the same line as the opening brace.

Reply:
fixed

Reply:
fixed

Reply:
I don't see a change

Reply:
Fixed.